### PR TITLE
Added `README.id.md` file for Indonesian translation

### PR DESCRIPTION
**Pull request by ardihikaru**
### Description

Explain why the courses you propose should be added.

### Pull request checklist

- [x] Changes added to [data.csv](https://github.com/SkalskiP/courses/blob/master/automation/data.csv). Read contribution [guidelines](https://github.com/SkalskiP/courses/blob/master/CONTRIBUTING.md) to learn more. (No changes to data.csv done)
- [x] Changes added to [README.md](https://github.com/SkalskiP/courses/blob/master/automation/data.csv) Read contribution [guidelines](https://github.com/SkalskiP/courses/blob/master/CONTRIBUTING.md) to learn more.
- [x] Pull request description added

### CODE REVIEW

File: README.id.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  
+</div>
+
+## 👋 Halo
+
+Repositori ini berisikan mengenai kumpulan informasi mengenai Kecerdasan Buatan (AI) dari berbagai sumber. Informasi yang disampaikan disini ditujukan untuk kaum pemula maupun yang sudah profesional dibidang ilmu Kecerdasan Buatan. 
+
+## 🚧 Bantu kami dalam berkontribusi
+
+Bahasa Inggris bukanlah satu-satunya bahasa di dunia. Ada banyak orang yang tidak bisa berbahasa inggris. Banyak pembuat konten tidak membuat konten mereka dalam bahasa Inggris. Oleh karena itu, kami ingin membuat kumpulan informasi yang terpisah yang dapat membantu orang yang ingin mempelajari Machine Learning (ML) dan tidak memiliki kemampuan untuk mengartikan bahasa dari konten tersebut. Bahasa Indonesia adalah bahasa selanjutnya yang ingin kami dukung. Ayo bantu kami dalam [membangun](https://github.com/SkalskiP/courses/discussions/22) kumpulan sumber informasi dalam Bahasa Indonesia.
```
Comment (SkalskiP) [inline README.id.md:17]: Could we update the discussion link from https://github.com/SkalskiP/courses/discussions/22 to https://github.com/SkalskiP/courses/discussions/34? The discussion you linked is about Hindi resources. I just created a separate one for Indonesian.

**Review by SkalskiP — CHANGES_REQUESTED** [review]

**Comment by SkalskiP** [comment]
> Should I update the other README files to add the Indonesian language selection as well

Could you add that link to the main `README.md` as well as README's for other languages? That would be so helpful. 🙏🏻

File: README.id.md (hunk 1 above)
Comment (ardihikaru) [reply README.id.md]: Hi. 
ah, thanks for noticing that one. I Updated the PR.

**Comment by ardihikaru** [comment]
> > Should I update the other README files to add the Indonesian language selection as well
> 
> Could you add that link to the main `README.md` as well as README's for other languages? That would be so helpful. 🙏🏻

Hi @SkalskiP I have updated the PR. Please re-review it.

**Review by SkalskiP — APPROVED** [review]

**Comment by SkalskiP** [comment]
Hi, @ardihikaru 👋🏻! Sorry for the late response. I was pretty busy. Looks great. Merging. Now we need to fill that README with links to excellent content! 🔥

**Pull request merged**